src/main/java/com/es/phoneshop/Cart/Cart.java

```diff
@@ -6,23 +6,40 @@
 

```

Comment:
I don't see payment functionality - from the lection:
 - Payment method selectbox: money, credit card
 - Place order button